### ISSUE TITLE
the void

### ISSUE THREAD

**Issue by jinshiawa** [FakeMan2332/MERRoomReplacement#5]
When I use MERRoomReplacement, the schematic it replaces becomes a void.But MER displays "Initializing schematic"

**Comment by FakeMan2332** [comment]
Hi! Please make sure your schematic in unity placed at correct cords.
- Move schematic in a room's folder, set cords and rotation to 0, 0, 0
- Correct positioning by moving children objects,** not a schematic root.**
- Save and compile.

**Comment by jinshiawa** [comment]
> 你好！请确保您的原理图统一放置在正确的电线上。
> 
> * 在房间文件夹中移动原理图，将电线和旋转设置为 0, 0, 0
> * 通过移动子对象而**不是原理图根来正确定位。**
> * 保存并编译。

No,I use "HczTestroomOld" schematic in MER discord.When I use old version,the schematic can replaces.But old version cannot start the round.

**Comment by jinshiawa** [comment]
> Hi! Please make sure your schematic in unity placed at correct cords.
> 
> * Move schematic in a room's folder, set cords and rotation to 0, 0, 0
> * Correct positioning by moving children objects, **not a schematic root.**
> * Save and compile.

No,I use "HczTestroomOld" schematic in MER discord.When I use old version,the schematic can replaces.But old version cannot start the round.

**Comment by FakeMan2332** [comment]
I recently used this schematic, everything were ok

**Comment by jinshiawa** [comment]
> Reference in new issue

Ok.Is it a conflict between plugins?

**Comment by FakeMan2332** [comment]
> his is debug is it right?

Everything is ok. It preventing 079's components from destroying.

**Comment by FakeMan2332** [comment]
Try to spawn a schematic by `mp cr shematic_name`. Schematic should spawn under a crosshair

**Comment by jinshiawa** [comment]
> Try to spawn a schematic by `mp cr shematic_name`. Schematic should spawn under a crosshair

I tried it can spawn.

**Comment by jinshiawa** [comment]
> > > Try to spawn a schematic by `mp cr shematic_name`. Schematic should spawn under a crosshair
> > 
> > 
> > So,what should I do?
> 
> I don't know, really. Everybody uses this feature without same issues. Try to use another schematic. Recheck entire schematic positions. Try to use offsets parameters.

Now I have a definite answer. The replaceroom function of this plugin cannot be used in conjunction with the original schematic in MER. Do you know what to do

**Comment by jinshiawa** [comment]
> > > Try to spawn a schematic by `mp cr shematic_name`. Schematic should spawn under a crosshair
> > 
> > 
> > So,what should I do?
> 
> I don't know, really. Everybody uses this feature without same issues. Try to use another schematic. Recheck entire schematic positions. Try to use offsets parameters.

![image](https://github.com/FakeMan2332/MERRoomReplacement/assets/140790979/6c7928b0-1ea3-43cb-a8d1-1a888fa6552f)
This is error

**Comment by FakeMan2332** [comment]
> > > > Try to spawn a schematic by `mp cr shematic_name`. Schematic should spawn under a crosshair
> > > 
> > > 
> > > So,what should I do?
> > 
> > 
> > I don't know, really. Everybody uses this feature without same issues. Try to use another schematic. Recheck entire schematic positions. Try to use offsets parameters.
> 
> Now I have a definite answer. The replaceroom function of this plugin cannot be used in conjunction with the original schematic in MER. Do you know what to do

wdym?

**Comment by jinshiawa** [comment]
> > > > > Try to spawn a schematic by `mp cr shematic_name`. Schematic should spawn under a crosshair
> > > > 
> > > > 
> > > > So,what should I do?
> > > 
> > > 
> > > I don't know, really. Everybody uses this feature without same issues. Try to use another schematic. Recheck entire schematic positions. Try to use offsets parameters.
> > 
> > 
> > Now I have a definite answer. The replaceroom function of this plugin cannot be used in conjunction with the original schematic in MER. Do you know what to do
> 
> wdym?

 What I mean is that the functionality of this plugin can't be used in conjunction with MER's schematic auto-generation feature

**Comment by FakeMan2332** [comment]
Ok, I will try fix this on a week
